publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -272,15 +273,36 @@ public void createRabbitMqConnection() throws RemRemPublishException {
             }
             factory.setConnectionTimeout(tcpTimeOut);
             rabbitConnection = factory.newConnection();
+            rabbitConnection.addShutdownListener(new ShutdownListener() {
+                @Override
+                public void shutdownCompleted(ShutdownSignalException cause) {
+                    log.debug("Connection Shutdown completed " + cause);
```

Comment:
We should probably use cause.getMessage(). We don't need the whole stacktrace.

Reply:
Ok